docs/security/simple-ldap-variables.md

```diff
@@ -1,4 +1,28 @@
-# Simple LDAP Variables
+# Simple LDAP authentication variables
+
+## Variables installed
+
+The installation adds the following variables:
+
+| Variable name | Description |
+|---|---|
+| authentication_ldap_simple_bind_base_dn | Base distinguished name |
+| authentication_ldap_simple_bind_root_dn | Root distinguished name |
+| authentication_ldap_simple_bind_root_dn_pwd | Password for the root distinguished name |
+| authentication_ldap_simple_ca_path | Absolute path of the certificate authority |
+| authentication_ldap_simple_group_role_mapping | LDAP group name |
```

Comment:
No, this variable contains a list of ldap group name - mysql rola pairs, in the format of

"<ldap_group>=<mysql_role>,<ldap_group2>=<mysql_role2>,..."

Reply:
updated